### ISSUE TITLE
Parameter missing (formData with fields: type string + type file)

### ISSUE THREAD

**Issue by z0mt3c** [swagger-api/swagger-ui#944]
In case of the following Path Item swagger-ui renders two input fields correctly:
- name: input (type text)
- file: input (type file)

But unfortunately the "name" parameter is missing in the generated request body. Tested with 2.1.6-M1. Or is something wrong about my schema?

Path Item
```json
"/test/fileUpload": {
    "post": {
    	"tags": ["api"],
    	"parameters": [{
    		"required": false,
    		"type": "string",
    		"name": "namaae",
    		"in": "formData"
    	}, {
    		"required": false,
    		"type": "file",
    		"name": "file",
    		"in": "formData"
    	}],
    	"consumes": ["multipart/form-data"],
    	"responses": {
    		"default": {}
    	},
    	"produces": ["application/json"]
    }
}
```

Request payload
```
------WebKitFormBoundaryspHivtbDRzAk29ls
Content-Disposition: form-data; name="file"; filename="test.png"
Content-Type: image/png
------WebKitFormBoundaryspHivtbDRzAk29ls--
```

**Comment by webron** [comment]
The definition looks fine, so that's a bug.

What generated your spec?

**Comment by z0mt3c** [comment]
hapi-swaggered@2.0.0-alpha.2 :-)

**Comment by fehguy** [comment]
Found the issue, fix is in 2.1.6-M1
